tests/common-tests.h

```diff
@@ -4260,6 +4279,97 @@ TEST_CASE_METHOD(common_tests, "String length", "[core][string][length]")
     CHECK(vout[2].length() == 20);
 }
 
+// Helper function used in two tests below.
+static std::string make_long_xml_string()
+{
+    std::string s;
+    s.reserve(10000);
```

Comment:
Is this arbitrary size or is there any rationale behind?

Reply:
The string constructed below is `200*26 + 12` bytes long, so I guess I could use its exact length instead, but I just lazily rounded up...

Reply:
If I use 'magic' numbers, I'd put a tiny comment

```
s.reserve(10000); // arbitrary large
```

That makes it clear to the reader that this is not a special number, documented anywhere or required by an (presumably) thrid-party API, etc. Less questions raised, simpler to comprehend.

Reply:
Maybe could use 3 constants for those 3 number in expression 200*26 + 12.